README.md

```diff
@@ -133,10 +134,38 @@ CycleTimeMs - Cycle time of the application.  All tags will be posted at this sp
 
 PostDuplicateTagValues - Controls when datapoints are logged.  If set to true, datapoints will always be cyclicly logged. If set to false, datapoints will only be logged on change of value.
 
+CommunicationType - This refers to if you are using OMF (OSIsoft message format). If you have PIWEBAPI 2019 or later, and you have OMF enabled, set this string to "omf". If you are using an older version of PIWEBAPI set the string to "piwebapi". Any other values will result in an error. for more information on OMF read the section here:  [OMF](#OMF-Support)
```

Comment:
the last sentence should start with a capital.

Reply:
oops

Reply:
i fixed this just now